Gemfile.lock

```diff
@@ -83,30 +83,31 @@ GEM
       rake (>= 10.4, < 14.0)
     apipie-rails (0.5.18)
       rails (>= 4.1)
-    arbre (1.2.1)
-      activesupport (>= 3.0.0)
-    ast (2.4.0)
+    arbre (1.3.0)
+      activesupport (>= 3.0.0, < 6.1)
+      ruby2_keywords (>= 0.0.2, < 1.0)
+    ast (2.4.1)
     aws-eventstream (1.1.0)
-    aws-partitions (1.326.0)
-    aws-sdk-core (3.98.0)
+    aws-partitions (1.381.0)
+    aws-sdk-core (3.109.1)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.33.0)
-      aws-sdk-core (~> 3, >= 3.71.0)
+    aws-sdk-kms (1.39.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.67.1)
-      aws-sdk-core (~> 3, >= 3.96.1)
+    aws-sdk-s3 (1.83.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.1.4)
-      aws-eventstream (~> 1.0, >= 1.0.2)
+    aws-sigv4 (1.2.2)
```

Comment:
### [aws-sigv4](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.1.4 → 1.2.2

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sigv4/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sigv4)_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.1.4...1.2.2).

These are the individual commits:
- (98a08af) [Version 1.2.0 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/98a08af6d16103eb642121dee410b14a57bd1caf)
- (90f8b27) [S3 client should send optional content type on copy object](https://github.com/aws/aws-sdk-ruby/commit/90f8b27a7b0be6ed7fae44d762c5e7154afad35c)
- (43e872b) [S3 object should be able to change content type on copy from](https://github.com/aws/aws-sdk-ruby/commit/43e872be2ac542bef6d431307ba7bf7e456127e0)
- (d288c8d) [Version 1.2.1 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/d288c8df3d842017b91f4346a42c0a3458ae1337)
- (216fa48) [domain scope to facilitate sharding](https://github.com/aws/aws-sdk-ruby/commit/216fa4836cfc7afef6084e747e9206f2df6452a2)
- (b6ee4ad) [Version 1.2.2 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/b6ee4ad5027403e0813b5c4ce85c7a5305c6a100)


</details>